index.html

```diff
@@ -3857,6 +3857,73 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media type used by this specification relies on JSON-LD, JSON-LD
+processing is not required to be performed, since JSON-LD is JSON. Some
```

Comment:
```suggestion
While the base media type used by this specification relies on JSON-LD, and
understanding of JSON-LD is a prerequisite, JSON-LD
processing is not required to be performed in certain cases, since JSON-LD is JSON. Some
```

Reply:
Building off @Sakurann's suggestion —
```suggestion
While the base media type used by this specification relies on JSON-LD, and
understanding of JSON-LD is a prerequisite, JSON-LD processing is not always
required; sometimes processing the JSON-LD as JSON &mdash; of which JSON-LD
is a subtype &mdash; is sufficient. Some
```

Reply:
`understanding of JSON-LD is a prerequisite` --> `an understanding of JSON-LD is a prerequisite`